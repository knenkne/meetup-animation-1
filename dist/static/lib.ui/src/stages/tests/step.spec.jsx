import React from 'react'
import renderer from 'react-test-renderer'

import { Step } from '../step'

describe('<Stages.Step />', () => {
    it('Рендерит заголовок', () => {
        const tree = renderer
            .create(<Step title="Заголовок"><span>{'Текст'}</span></Step>)
            .toJSON()

        expect(tree).toMatchSnapshot()
    })

    it('Имеет класс tooltip если передан mode', () => {
        const tree = renderer
            .create(<Step title="Заголовок" mode="progress"><span>{'Текст'}</span></Step>)
            .toJSON()

        expect(tree).toMatchSnapshot()
    })
})
