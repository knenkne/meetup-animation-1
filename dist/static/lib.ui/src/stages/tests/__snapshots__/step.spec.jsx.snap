// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stages.Step /> Имеет класс tooltip если передан mode 1`] = `
<li
  className="step current"
>
  <div
    className="tooltip"
  />
  <div
    className="marker"
  >
    <span
      className="icon checkMark"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"11\\" height=\\"9\\" viewBox=\\"0 0 11 9\\"><path fill=\\"none\\" stroke=\\"#FFF\\" stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" stroke-width=\\"2\\" d=\\"M1 4.948L3.899 8 10 1\\"/></svg>",
        }
      }
      data-unit="icon"
      onClick={[Function]}
      onError={[Function]}
    />
    <span
      className="markerIndex"
    />
  </div>
  <div
    className="title"
  >
    Заголовок
  </div>
</li>
`;

exports[`<Stages.Step /> Рендерит заголовок 1`] = `
<li
  className="step"
>
  <div
    className="marker"
  >
    <span
      className="icon checkMark"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"11\\" height=\\"9\\" viewBox=\\"0 0 11 9\\"><path fill=\\"none\\" stroke=\\"#FFF\\" stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" stroke-width=\\"2\\" d=\\"M1 4.948L3.899 8 10 1\\"/></svg>",
        }
      }
      data-unit="icon"
      onClick={[Function]}
      onError={[Function]}
    />
    <span
      className="markerIndex"
    />
  </div>
  <div
    className="title"
  >
    Заголовок
  </div>
</li>
`;
